tests/unit/handlers/test__helpers.py

```diff
@@ -89,7 +140,11 @@ def test_no_context_header(self):
 
         middleware = request.RequestMiddleware(None)
         middleware.process_request(django_request)
-        trace_id = self._call_fut()
+        http_request, trace_id = self._call_fut()
+        self.assertEqual(http_request["request_method"], "GET")
+        self.assertEqual(set(http_request.keys()), set(_HTTP_REQUEST_FIELDS))
+        for field in _HTTP_REQUEST_FIELDS:
+            self.assertTrue(field in http_request)
```

Comment:
not familiar with the test framework in python - is this method overload printing the message to add context of what is expected vs actual?

E.g.
instead of:
```
expected: "abc", actual: "xyz"
```
it should be 
``` 
expected:"abc", actual:"xyz" **for <<field>>**
```

Reply:
Good point, I should have added a message for the test here. 

But I actually realized this assertion is redundant with the one above it, and it already gives a clear error. So I pulled this one out